plugins/plugin-java/che-plugin-java-ext-lang-client/src/main/java/org/eclipse/che/ide/ext/java/client/service/CustomNotificationReceiver.java

```diff
@@ -113,50 +98,15 @@ private void subscribe(RequestTransmitter transmitter) {
         .sendAndSkipResult();
   }
 
-  private Promise<Container> updateProject(String project) {
-    return appContext
-        .getWorkspaceRoot()
-        .getContainer(project)
-        .thenPromise(
-            optContainer -> {
-              if (optContainer.isPresent()) {
-                Container container = optContainer.get();
-                return projectService
-                    .getProject(Path.valueOf(project))
-                    .thenPromise(
-                        projectConfigDto -> {
-                          return projectService
```

Comment:
Fields `projectService` and `promises` become unused in this class and can be removed as well.